deid/config/utils.py

```diff
@@ -258,9 +258,40 @@ def parse_filter_group(spec):
     return members
 
 
+def derive_ctp_coordinate(raw):
+    """
+    Derive a ctp coordinate from a raw (comma separated) string.
+
+    A ctp coordinate is:
+    - the horizontal component of the left side
+    - the vertical component of the top
+    - the width
+    - the height
+
+    And we need to translate that into (xmin, ymin, xmax, ymax)
+    This largely means given that we have:
+      (--> (xmin), ^ (ymin), width, height)
+    This translates to (xmin, ymin, xmin+width, ymin+height)
+    Note that the first two values have no change, and the latter two are
+    derived by adding the width or height to the appropriate dimension.
+    """
+    # Cut out early if we have an "all" to indicate the entire image
+    if raw == "all":
+        return raw
+    new_coordinate = [int(x) for x in raw.split(",") if x]
+
+    # Cut out early for malformed coordinate
+    if len(new_coordinate) != 4:
+        bot.exit("Coordinates are expected to have length of 4, found %s" % raw)
+    xmin, ymin, width, height = new_coordinate
+
+    # Translate CTP coordinate to the convention we use
+    return [xmin, ymin, xmin + width, ymin + height]
```

Comment:
There was a little bug with this return.  Downstream locations expect coordinates to be a string comma separated list, not an actual list of ints. I've commited a fix in with PR #237.